plugins/scaffolder/src/components/MultistepJsonForm/ReviewStep.tsx

```diff
@@ -25,36 +25,38 @@ export function getReviewData(
   uiSchemas: UiSchema[],
```

Comment:
Wow, this diff is really unnecessarily noisy. A `git diff --ignore-all-space` shows that the only material changes in this iteration are like five lines in the beginning of the loop. The rest is whitespace from shifting indentation.